kevm-pyk/src/kevm_pyk/kproj/evm-semantics/lemmas/int-simplification.k

```diff
@@ -83,6 +83,15 @@ module INT-SIMPLIFICATION-HASKELL [symbolic]
     rule { A -Int B #Equals C } => { A #Equals C +Int B } [concrete(B, C), symbolic(A), simplification(45), comm]
     rule { A -Int B #Equals C } => { A -Int C #Equals B } [concrete(A, C), symbolic(B), simplification(45), comm]
 
+    rule A +Int B  <Int C +Int D => A -Int C  <Int D -Int B [concrete(B, D), symbolic(A, C), simplification(45)]
+    rule A +Int B <=Int C +Int D => A -Int C <=Int D -Int B [concrete(B, D), symbolic(A, C), simplification(45)]
+    rule A +Int B  >Int C +Int D => A -Int C  >Int D -Int B [concrete(B, D), symbolic(A, C), simplification(45)]
+    rule A +Int B >=Int C +Int D => A -Int C >=Int D -Int B [concrete(B, D), symbolic(A, C), simplification(45)]
+
+    rule A +Int B ==Int C +Int D => A -Int C ==Int D -Int B [concrete(B, D), symbolic(A, C), simplification(45)]
+
+    rule { A +Int B #Equals C +Int D } => { A -Int C #Equals D -Int B } [concrete(B, D), symbolic(A, C), simplification(45), comm]
```

Comment:
Dose this rule need the `comm` attribute? `comm` attribute causes the frontend to duplicate the rule with the arguments of the top-level symbol flipped. The rule that the frontend will create will be identical to this one anyway, so it just adds to th erules to try I think.

Reply:
Yes, this rule is fully commutative already, I will remove the attribute.